### ISSUE TITLE
CLI: add-locale doesn't support hyphens

### ISSUE THREAD

**Issue by adamgruber** [lingui/js-lingui#207]
I want to use `en-US` as my locale but the cli says it is unknown. Looking at the code, it seems like it would be a minor change to add support.

This RegExp
```js
// packages/cli/src/api/formats/utils/locales.js
export const localeRe = new RegExp(/(([a-z]+)(_([a-zA-Z]+))?)(\\|\/)?/)
```
could be changed to
```js
export const localeRe = new RegExp(/(([a-z]+)([_-]([a-zA-Z]+))?)(\\|\/)?/)
```

I can open a PR for the change but wasn't sure if there was a specific reason not to support hyphens?

**Comment by tricoder42** [comment]
The only reason is that I missed this :/ Feel free to send a PR! Very appreciated!

**Comment by tricoder42** [comment]
Thanks @adamgruber, fixed in **2.0.8**!
